src/formio/templates/component.ejs

```diff
@@ -6,5 +6,5 @@
     {% if (ctx.visible) { %}
       {{ctx.children}}
     {% } %}
-  <div ref="messageContainer" class="{{ctx.ofPrefix}}errors" aria-live="{{ctx.component.validateOn === 'blur' ? 'assertive' : 'polite'}}"></div>
+  <div ref="messageContainer" class="{{ctx.ofPrefix}}errors" aria-live="{{ctx.component.validateOn === 'blur' ? 'assertive' : 'polite'}}" aria-atomic="true"></div>
```

Comment:
Ik denk dat `aria-live="polite"` alleen is voor onBlur, en anders helemaal geen `aria-live`.

Reply:
De `messageContainer`, of beter nog: elke error, moet eigenlijk ook nog een `id` hebben die gekoppeld is aan de form input met `aria-describedby`.

Reply:
Correctie:
Gebruik `aria-live="assertive"` voor foutmeldingen, niet `polite`. `polite` is voor statusmeldingen zoals "Nog 8 tekens tot de limiet."

Reply:
Voor styling hebben we de `utrecht-form-field-description utrecht-form-field-description--invalid` class name.
https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-form-field-description--invalid

Reply:
Punt is dat we hier 1 component hebben dat meerdere soorten alerts kan geven, en soms al reageert tijdens typen. Bijvoorbeeld: Telefoonnumer en e-mail  - daar vindt dan eigenlijk wel een mix plaats van 'assertive' en 'polite'.
Maar: Role mag je niet combineren met aria-live: "_adding both aria-live and role="alert" causes double speaking issues in VoiceOver on iOS..._"
Dus dan zou het nu tijdelijk zo kunnen worden:
`<div ref="messageContainer" class="{{ctx.ofPrefix}}errors" role="alert" aria-describedby="{{ctx.id}}"></div>`
(aria-atomic="true" zit dan automatisch in role=alert).

Reply:
> Voor styling hebben we de `utrecht-form-field-description utrecht-form-field-description--invalid` class name. https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-form-field-description--invalid

I've refactored our formio templates and custom templates to make use of this component + added the design tokens to our theme.